java/ql/src/experimental/Security/CWE/CWE-200/SensitiveAndroidFileLeak.ql

```diff
@@ -0,0 +1,77 @@
+/**
+ * @name Leaking sensitive Android file
+ * @description Getting file intent from user input without path validation could leak arbitrary
+ *              Android configuration file and sensitive user data.
+ * @kind path-problem
+ * @id java/sensitive-android-file-leak
```

Comment:
Missing `@severity`

Reply:
I never specified <code>@severity</code> in queries before - I think the CodeQL evaluation team will add it. Shall I specify <code>@problem.severity error</code>? Please advise. Thanks.

Reply:
Added <code>@problem.severity warning</code>